File: 8kyu/grasshopper-grade-book/grasshopper-grade-book.py
def get_grade(s1: int, s2: int, s3: int) -> str:
    avg = sum([s1, s2, s3]) / 3
    if avg >= 90:
        return 'A'
    elif avg >= 80:
        return 'B'
    elif avg >= 70:
        return 'C'
    elif avg >= 60:
        return 'D'
    else:
        return 'F'

File: 8kyu/grasshopper-grade-book/grasshopper-grade-book.go
package main

import "fmt"

func GetGrade(a,b,c int) rune {
	grades := []int{a,b,c}

	switch avg := (grades[0] + grades[1] + grades[2]) / 3; {
		case avg < 60: 
    return 'F'
    case avg < 70: 
    return 'D'
    case avg < 80: 
    return 'C'
    case avg < 90: 
    return 'B'
    default: 
    return 'A'
	}
}

func main() {
	fmt.Println(GetGrade(95,90,93))
}